.changeset/rotten-peaches-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+'@eth-optimism/l2geth': patch
```

Comment:
It pains me that this is considered a patch but I presume that it is only when we reach 1.0 that we will move away from patches being backwards incompatible?

Reply:
It's `minor`